addons/website_event/tests/test_website_event.py

```diff
@@ -119,3 +119,34 @@ def test_sitemap(self):
         self.assertTrue('/event/event-0' in resp.text, 'Published events must be present in the sitemap')
         self.assertTrue('/event/event-1' in resp.text, 'Published events must be present in the sitemap')
         self.assertFalse('/event/event-2' in resp.text, 'Unpublished events must not be present in the sitemap')
+
+class TestMail(MailCommon):
```

Comment:
Could probably belong to its own test file, like 'test_event_mail.py' with mail related stuff we might want to add :)